docs/03-privacy/privacy-incident-response.md

```diff
@@ -0,0 +1,929 @@
+# Bixal Solutions Privacy Incident Response Plan (IRP)
+
+## Applicability
+
+**Note:**  This Incident Response Plan applies to systems for which Bixal Solutions and third party
```

Comment:
The first sentence of the **Note** section seems to be missing a word or two. Are we referring to "systems that Bixal Solutions and third party contractors ***use***, ***interact with*** or ***come across***, maybe?